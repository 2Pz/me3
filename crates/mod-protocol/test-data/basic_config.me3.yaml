profileVersion: v1
natives:
  - enabled: true
    optional: true
    path: "./mod.dll"

packages:
  - id: my-mod
    source: "mod/"