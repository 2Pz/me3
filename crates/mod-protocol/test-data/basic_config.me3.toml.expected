V1(
    ModProfileV1 {
        natives: [
            Native {
                path: "my_native.dll",
                optional: true,
                enabled: true,
                load_before: [],
                load_after: [],
                initializer: None,
                finalizer: None,
            },
        ],
        packages: [
            Package {
                id: "my-mod",
                source: PackageSource(
                    "mod/",
                ),
                load_after: [],
                load_before: [],
            },
        ],
    },
)
